manifests/eureka-example/deployment.yaml

```diff
@@ -0,0 +1,28 @@
+name: eureka-example
+toolchainRegion: us-east-2
```

Comment:
Is the blogpost targetting `us-east-2` region?

Reply:
Will change to `us-east-1`